dataland-frontend/src/components/resources/dataTable/MultiLayerDataTableBody.vue

```diff
@@ -110,6 +125,10 @@ function expandSectionsOnPageLoad(): void {
   }
 }
 
+const columnWidthStyle = computed(() => {
```

Comment:
Won't this look very condensed if there are datasets for many reporting periods? Don't we have something like sideways scrolling?

Reply:
TODO for me